class MyError(ZeroDivisionError):
    # Я так и не понял что значит "Создать собственный класс-исключение, обрабатывающий ситуацию деления на ноль."
    # Классs-исключения не обрабатывает ситуацию а лишь создают собственныеисключение. Обрабатывает его уже программа
    def __str__(self):
        return 'Деление на 0'


a = int(input('Введи число '))
b = int(input('Введи второе число '))
try:
    if b == 0:
        raise MyError
except MyError:
    print('Деление на 0')
else:
    print(a / b)
